src/Notifications.h

```diff
@@ -7,6 +7,11 @@
 #include "Strings.h"
 
 constexpr uint32 cNotifCallbackID = 0x8001; //  ie. WM_APP + 1
+// Not used yet but could be used as an ID mask to dispatch events.
+[[maybe_unused]]
+constexpr uint32 cNotifMenuId = 0x8100;
```

Comment:
Does that value mean something specific? Or it's just a value that you picked?

Reply:
I just used `0x81xx` as a pseudo-namespace where the least significant byte would be the notification menu's command ID. The upper byte would just identifying the context and masking would be easier to think about.

Just some forward thinking on the spot, not really useful as is *for now*.